src/views/Editors.vue

```diff
@@ -50,20 +48,38 @@ export default {
     AppTitledList,
     EditorsIntro
   },
+  data () {
+    return {
+      asideListRoleValue: '總編'
+    }
+  },
   computed: {
+    asideListRoleKey () {
+      return _.find(ROLE_MAP, { value: this.asideListRoleValue }).key
+    },
     customEditors () {
       return _.get(this.$store, 'state.customEditors.items', [])
+    },
+    asideListMembers () {
+      return _.get(this.$store, `state.publicMembers[${this.asideListRoleValue}].items`, [])
     }
   },
   beforeMount () {
     Promise.all([
-      getCustomEditors(this.$store, {})
+      getMembersPublic(this.$store, {
+        custom_editor: true
+      }),
+      getMembersPublic(this.$store, {
+        role: this.asideListRoleKey
+      })
```

Comment:
so we call the api twice?

Reply:
Since custom editor not only is a guest editor, but also maybe another role like admin and editor. So I need to request twice because I can't join query together?